.github/CODEOWNERS

```diff
@@ -44,9 +44,10 @@
 /filebeat/                               @wk8 jean@tripping.com
 /finout/                                 @roiravhon support@finout.io
 /flagsmith/                              @benjarom3 support@flagsmith.com
-/fluentbit/                             @zhenyami
+/fluentbit/                              @zhenyami
```

Comment:
Can you rebase from master? I think this was fixed in https://github.com/DataDog/integrations-extras/pull/1342/files

Reply:
Rebased